docs/using_eessi/eessi_container.md

```diff
@@ -0,0 +1,165 @@
+# Using the EESSI container
+
+This page provides all information and guides you through several examples
+illustrating to access the EESSI software stack via a small container provided by
+EESSI. The container only contains a minimal number tools such as the CernVM-FS
+client and some configuration data.
+
+### Prerequisites
+
+- Singularity 3.7.x or newer _or_ Apptainer x.y.z or newer (any version with
+  support for `--fusemount` should be sufficient)
+- `git` available on your system
+
+### Preparation
+
+Clone the [`EESSI/software-layer`](https://github.com/EESSI/software-layer.git)
+repository to a directory `eessi_container` by running the command
+```bash
+git clone https://github.com/EESSI/software-layer.git eessi_container
+```
+Change directory
+```bash
+cd eessi_container
+```
+
+## Basic scenarios
+
+### Get access to EESSI
+
+Simply run the command (from the directory you have changed into above)
+```bash
+./eessi_container.sh
+```
+_(Note, the first run may take a bit longer because the container image is
+downloaded and converted.)_
+
+You should see output like
+```
+Using /tmp/eessi.RC6kYEjmSj as tmp storage (add '--resume /tmp/eessi.RC6kYEjmSj' to resume where this session ended).
+Launching container with command (next line):
+singularity shell --fusemount container:cvmfs2 pilot.eessi-hpc.org /cvmfs/pilot.eessi-hpc.org docker://ghcr.io/eessi/build-node:debian11
+CernVM-FS: pre-mounted on file descriptor 3
+Singularity> CernVM-FS: loading Fuse module... done
+fuse: failed to clone device fd: Inappropriate ioctl for device
+fuse: trying to continue without -o clone_fd.
+
+Singularity>
+```
+_(Note, you may have to press enter to clearly see the prompt as some messages
```

Comment:
For notes, you can use:

```
!!! note
    Example
```

Reply:
Fixed by https://github.com/EESSI/docs/pull/100/commits/0eae16fe58096e7b995d7e7d2ddc3d9208fc2001

Also changed most _Note ..._ accordingly.